lib/gamnit/network/server.nit

```diff
@@ -39,7 +39,7 @@
 #
 #     # `accept_clients` in non-blocking,
 #     # sleep before tying again, or do something else.
-#     nanosleep(0, 50000000)
+#     0.5.sleep
```

Comment:
You should add parentheses around the number to make this less ambiguous for the user.